edi_webservice_oca/tests/test_send.py

```diff
@@ -88,3 +106,19 @@ def test_component_send(self):
             responses.calls[0].request.headers["Content-Type"], "application/xml"
         )
         self.assertEqual(responses.calls[0].request.body, "This is a simple file")
+
+    @responses.activate
+    def test_compenent_send_by_end_user(self):
```

Comment:
this test could be skipped: use the user in the test above. If the avg is able to run that, admins can do it for sure :wink:

Reply:
done !